### PR TITLE
Logout Mobile Button

### PR DESCRIPTION
**Pull request by etsman11**
I took out the login-url class, I also deleted the code that looked for current user and replaced the login option with the user name. That should now replaced with the logout button.

### CODE REVIEW

File: wp-content/themes/v2w/header.php [hunk 1]
```diff
@@ -97,13 +97,9 @@
 				<ul class="top">
 					<li class="first"><a href="#" class="menu-trigger" id="mobile-menu-trigger"></a></li>
 					<li class="mid"><a href="<?php echo get_option('siteurl'); ?>" class="logo"></a></li>
-					<li class="last login-url"><!-- /Added Logout Button -->
+					<li class="last"><!-- /Added Logout Button -->
 						<?php if( is_user_logged_in() ) : ?>
-							<?php $current_user = wp_get_current_user(); ?>
-							<a href="<?php echo get_option('siteurl'); ?>/shop/account"><?php echo $current_user->user_login; ?></a>
-							<li class="logout-url">
 								<a href="<?php echo get_option('siteurl'); ?>/shop/account/?logout" class="logout">Logout</a>
```
Comment (MikeEstes) [inline wp-content/themes/v2w/header.php]: Formatting: Remove Tab

File: wp-content/themes/v2w/header.php [hunk 2]
```diff
@@ -131,4 +127,4 @@
 			</div>
 
 		</div>
-	</header>
\ No newline at end of file
+	</header>Logout
```
Comment (MikeEstes) [inline wp-content/themes/v2w/header.php]: Remove extra word.

**Comment by etsman11** [comment]
I took out the extra tab indentation and the "Logout" at the end of </header>

**Comment by MikeEstes** [comment]
+1

**Pull request merged**